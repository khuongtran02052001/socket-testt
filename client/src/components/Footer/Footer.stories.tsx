import React from 'react';

import { Meta, Story } from '@storybook/react';

import { Footer, FooterProps } from './Footer';

const story: Meta = {
  title: 'Components/Footer',
  component: Footer,
};

export const Default: Story = (props: FooterProps) => {
  return <Footer {...props}></Footer>;
};

Default.args = {};

Default.parameters = {
  jest: ['Header.test.tsx'],
  layout: 'fullscreen',
};

export default story;
