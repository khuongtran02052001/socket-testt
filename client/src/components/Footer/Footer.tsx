import React from 'react';

export interface FooterProps {
  className?: string;
}
export const Footer: React.FC<FooterProps> = (props) => {
  return (
    <div className="bg-gray-300 text-center text-black p-10" {...props}>
      <div className="flex justify-center flex-col ">
        <h1 className="hover:text-red-400 cursor-pointer font-bold">
          DEV Community
        </h1>
        <p>
          A constructive and inclusive social network for software developers.
          With you every step of your journey.
        </p>
      </div>
      <div className="flex justify-center p-7">
        Built on
        <p className="px-1 hover:text-red-400 cursor-pointer font-bold">
          Forem
        </p>
        <p>-- the</p>
        <p className="px-1 hover:text-red-400 cursor-pointer font-bold">
          open source
        </p>
        <p>software that powers </p>
        <p className="px-1 hover:text-red-400 cursor-pointer font-bold">DEV</p>
        <p> and other .</p>
      </div>
      <div className="flex justify-center">
        Made with love and
        <p className="px-1 hover:text-red-400 cursor-pointer font-bold">
          {' '}
          Ruby on Rails.{' '}
        </p>
        <p> DEV Community Â© 2016 - 2021.</p>
      </div>
    </div>
  );
};
