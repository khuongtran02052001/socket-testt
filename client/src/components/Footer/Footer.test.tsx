import React from 'react';

import renderer from 'react-test-renderer';

import { Footer } from './Footer';

it('renders correctly', () => {
  const tree = renderer.create(<Footer></Footer>).toJSON();
  expect(tree).toMatchSnapshot();
});
