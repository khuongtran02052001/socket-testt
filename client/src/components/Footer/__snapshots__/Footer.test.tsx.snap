// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-gray-300 text-center text-black p-10"
>
  <div
    className="flex justify-center flex-col "
  >
    <h1
      className="hover:text-red-400 cursor-pointer font-bold"
    >
      DEV Community
    </h1>
    <p>
      A constructive and inclusive social network for software developers. With you every step of your journey.
    </p>
  </div>
  <div
    className="flex justify-center p-7"
  >
    Built on
    <p
      className="px-1 hover:text-red-400 cursor-pointer font-bold"
    >
      Forem
    </p>
    <p>
      -- the
    </p>
    <p
      className="px-1 hover:text-red-400 cursor-pointer font-bold"
    >
      open source
    </p>
    <p>
      software that powers 
    </p>
    <p
      className="px-1 hover:text-red-400 cursor-pointer font-bold"
    >
      DEV
    </p>
    <p>
       and other inclusive communities.
    </p>
  </div>
  <div
    className="flex justify-center"
  >
    Made with love and
    <p
      className="px-1 hover:text-red-400 cursor-pointer font-bold"
    >
       
      Ruby on Rails.
       
    </p>
    <p>
       DEV Community Â© 2016 - 2021.
    </p>
  </div>
</div>
`;
